src/components/DocumentList.tsx

```
@@ -394,25 +390,23 @@ const DocumentList: React.FC<DocumentListProps> = ({
-                      w="100%"
-                    >
-                      <Text fontWeight="bold" color="red.600" fontSize="md">
-                        {err.raw}
-                      </Text>
-                      <Text color="gray.800" fontSize="sm">
-                        {err.error}
-                      </Text>
-                    </VStack>
-                  )
-                )}
+                {errorModalDoc.verificationErrors.map((err: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)